### PR TITLE
fix(ui5-side-navigation): fixed accessibility roles and attributes when Side Navigation is collapsed

### PR DESCRIPTION
**Pull request by LidiyaGeorgieva**
PartOf: #5827

### CODE REVIEW

File: packages/fiori/src/SideNavigation.hbs [hunk 1]
```diff
@@ -19,6 +20,7 @@
 					?has-children="{{this.item.items.length}}"
 					?expanded="{{this.item.expanded}}"
 					?selected="{{this.selected}}"
+					aria-roledescription="{{this.item.ariaRoleDescNavigationListItem}}"
```
Comment (TeodorTaushanov) [inline packages/fiori/src/SideNavigation.hbs:23]: roledescription="{{this.ariaRoleDescNavigationListItem}}"

File: packages/fiori/src/SideNavigationItem.js [hunk 2]
```diff
@@ -129,6 +134,17 @@ class SideNavigationItem extends UI5Element {
 	get _tooltip() {
 		return this.title || this.text;
 	}
+
+	get ariaRoleDescNavigationListItem() {
```
Comment (TeodorTaushanov) [inline packages/fiori/src/SideNavigationItem.js:138]: Move this to the SideNavigation class as it depends on the "collapsed" property of the whole SideNavigation.

File: packages/fiori/src/SideNavigationSubItem.js [hunk 3]
```diff
@@ -87,6 +92,17 @@ class SideNavigationSubItem extends UI5Element {
 	get _tooltip() {
 		return this.title || this.text;
 	}
+
+	get ariaRoleDescNavigationListItem() {
```
Comment (TeodorTaushanov) [inline packages/fiori/src/SideNavigationSubItem.js:96]: Same as the SideNavigationListItem

File: packages/fiori/src/SideNavigationItemPopoverContent.hbs [hunk 4]
```diff
@@ -1,20 +1,34 @@
...
 		mode="None"
+		aria-roledescription={{ariaRoleDescPopoverNavigationList}}
+		role="tree"
 		@ui5-item-click="{{handleListItemClick}}"
 	>
 		<ui5-li
 			title="{{_popoverContent.mainItem._tooltip}}"
+			role="treeitem"
+			aria-level="1"
+			ria-checked="{{this.selected}}"
```
Comment (TeodorTaushanov) [inline packages/fiori/src/SideNavigationItemPopoverContent.hbs:19]: aria-selected="{{this.selected}}"

File: packages/fiori/src/SideNavigationItemPopoverContent.hbs [hunk 5]
```diff
@@ -1,20 +1,34 @@
...
 			?selected="{{_popoverContent.mainItemSelected}}"
 			.associatedItem="{{_popoverContent.mainItem}}"
 		>{{_popoverContent.mainItem.text}}</ui5-li>
 
 		{{#each _popoverContent.subItems}}
 			<ui5-li
 				title="{{this._tooltip}}"
+				role="treeitem"
+				aria-level="2"
+				aria-checked="{{this.selected}}"
```
Comment (TeodorTaushanov) [inline packages/fiori/src/SideNavigationItemPopoverContent.hbs:30]: aria-selected="{{this.selected}}"

**Review by TeodorTaushanov — CHANGES_REQUESTED** [review]

File: packages/main/src/ListItem.js [hunk 6]
```diff
@@ -105,11 +118,19 @@ const metadata = {
...
 
-		_ariaHasPopup: {
+		/**
+		 * Defines the availability and type of interactive popup element that can be triggered by the component on which the property is set.
+		 * @type String
+		 * @since 1.10.0
+		 * @protected
+		 */
+		ariaHaspopup: {
 			type: String,
```
Comment (dobrinyonkov) [inline packages/main/src/ListItem.js:128]: Use the `HasPopup` enum.

**Review by dobrinyonkov — APPROVED** [review]

**Review by dobrinyonkov — APPROVED** [review]

**Review by TeodorTaushanov — APPROVED** [review]

**Pull request merged**